File: 351002/Mikhalkov/Distcomp.Shared/Models/Note.cs
using System.Text.Json.Serialization;

namespace Distcomp.Shared.Models
{
    public class Note
    {
        [JsonPropertyName("country")]
        public string Country { get; set; } = "BY";
        [JsonPropertyName("issueId")]
        public long IssueId { get; set; }
        [JsonPropertyName("id")]
        public long Id { get; set; }
        [JsonPropertyName("content")]
        public string Content { get; set; } = string.Empty;
        [JsonPropertyName("state")]
        public NoteState State { get; set; } = NoteState.PENDING;
    }
}

File: 351003/Matelenok/discussion/src/main/java/by/distcomp/app/model/Note.java
package by.distcomp.app.model;

import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import lombok.Getter;
import lombok.Setter;
import org.springframework.data.cassandra.core.mapping.*;

import java.time.Instant;

@Table("tbl_note")
@Getter @Setter
public class Note {

    @PrimaryKey
    private Long id;

    @Column("article_id")
    @Indexed
    private Long articleId;

    @Column("content")
    private String content;

    @Column("created")
    private Instant created = Instant.now();
    @Enumerated(EnumType.STRING)
    private NoteState state = NoteState.PENDING;
}